4Gewinnt.py

```diff
@@ -0,0 +1,13 @@
+from MainGame import MainGame
+from Board import Board
+
+def main() -> None:
+    current_amount_of_rounds = 0
+    game = MainGame()
+    board = Board()
+    board.show_board(board, current_amount_of_rounds)
+    current_amount_of_rounds += 1
+    game.play(board, current_amount_of_rounds)
```

Comment:
```suggestion
    current_amount_of_rounds = 0
    game = MainGame()
    board = Board()
    board.show_board(board, current_amount_of_rounds)
    current_amount_of_rounds += 1
    game.play(board, current_amount_of_rounds)
```
Es ist verwirrend, warum die Methode `show_board` die aktuelle Rundenanzahl benötigen sollte

Reply:
I just wanted to implement it into show_board because I wanted to prevent to add in multiple prints because of readability and cleaner code